### DISCUSSION TITLE
Parasitic manager not opening after reboot

### DISCUSSION THREAD

**Discussion by elswerky** [mywalkb/LSPosed_mod#97] [Q&A]
### Steps to reproduce

Hello bro 
I have an issue not related to your mod specifically but for lsposed and occur with your mod too in samsung devices like tab S6 lite 
Tab S7 plus 
With latest software updates,  lspsoed parasitic manager don't open after reboot 
And I can't open lsposed manager unless I install lsposed apk from store 

Teh issue I can no longer use xposed module I always use like xposed edge module 

Please if you can help me 
Lsposed is discounted and you are tye only hope I have to fix this issue or help me 

If you can contact me at telegram at @elswerky  to provide you with logs or answers about this issue 

Thanks bro 
You are my only hope 

### Expected behaviour

Then parasitic manager open after reboot without issue 

### Actual behaviour

Parasitic manager not opening as if opening shortcut for something not found 

### Xposed Module List

```shell
Only exposed edge
```

### Magisk Module List

```shell
Only lsposed and magisk bootloop protect
```

### LSPosed version

1.9.3

### Android version

13

### Magisk version

26.3

### Riru version

Don't use

### Version requirement

- [X] I am using latest debug CI version of LSPosed and enable verbose log

### Logs

Jhrhr
[log.log (1).txt](https://github.com/mywalkb/LSPosed_mod/files/15233123/log.log.1.txt)
[log.log.txt](https://github.com/mywalkb/LSPosed_mod/files/15233124/log.log.txt)

**Comment by mywalkb** [comment]
No debug log.
In the latest versions of android the parasitic manager is giving issues, that's why I decided to make the parasitic manager optional and put the random name of the manager as magisk. It is still in development.
Could you try giving root privileges in magisk to the com.android.shell package?
Someone solved it like this.

**Comment by elswerky** [comment]
> No debug log. In the latest versions of android the parasitic manager is giving issues, that's why I decided to make the parasitic manager optional and put the random name of the manager as magisk. It is still in development. Could you try giving root privileges in magisk to the com.android.shell package? Someone solved it like this.

How to give root privileges 
I don't know how to do this 
And in superuser in magisk delta (kitsune manager ) I don't all apps that asked for root ingave permissions 

But don't know how to give privileges to com.android.shell 

Also am using  your mod v1.9.3 zygisk bit not debug version and don't know where is the debug version 


Places if you have telegram search me at @elswerky for rapid and fast access with you please

**Comment by elswerky** [comment]
> No debug log. In the latest versions of android the parasitic manager is giving issues, that's why I decided to make the parasitic manager optional and put the random name of the manager as magisk. It is still in development. Could you try giving root privileges in magisk to the com.android.shell package? Someone solved it like this.

Com.android.shell do8appear un magsik in apps asking for root so how to enable it as you said 


Plus 
You said you named parasitic manager to magisk but this isn't the case with me  
Please contact me at telegram for fast access please

**Comment by elswerky** [comment]
> No debug log. In the latest versions of android the parasitic manager is giving issues, that's why I decided to make the parasitic manager optional and put the random name of the manager as magisk. It is still in development. Could you try giving root privileges in magisk to the com.android.shell package? Someone solved it like this.

Bro 
You didn't reply me 
Com.android.shell not appearing in magisk  to enable root for it 
Also you say manager you named magisk , I don't see this  Here

**Comment by elswerky** [comment]
@mywalkb 
Bro at least try to fund a way to make xposed edge module qoek again on latest samsung softwares 
This is the main issue for me on latest Samsung software on some devices 

Xposed edge module is activated but gestures not working  although it is working with me on other smasung devices

**Comment by mywalkb** [comment]
Sorry for the delay, for give permission to com.android.shell connect with `adb shell` and switch user to root, you should receive a popup for grant access.
For debug version you can find them [here](https://github.com/mywalkb/LSPosed_mod/actions/workflows/core.yml?query=branch%3Amain)

**Comment by elswerky** [comment]
@mywalkb o grant permession for com.android.shell and now appear in magisk

What should I do then ??

**Comment by mywalkb** [comment]
> What should I do then ??

try to open manger from notification.

**Comment by elswerky** [comment]
> > What should I do then ??
> 
> try to open manger from notification.

I tried and it opened 
But it was opening from notification before 

It is the shortcut manager that I create that don't work after I reboot device

**Comment by elswerky** [comment]
Bro 
Sorry but I have to leave home today and will leave tablet at home , so I can't have access to it to test you ask me to do tonfix the issue 
Before I asked you for any social media contact for fast and easy contact to test all what you will ask me at a ssession 
But here I have to wait for a day or half a day for your reply and today I won have access to test 

BTW,  the issue not in your mod file 
If is lsposed issue with newer Samsung software updates of latest few months 
Where lsposed shortcut don't open after reboot.
.also the module I root for it , xposed edge is activated but not working as if can't hook system

**Comment by mywalkb** [comment]
> BTW, the issue not in your mod file

Ok I move to discussion.
